modules/tensor_mechanics/test/tests/uel/reference.i

```diff
@@ -0,0 +1,130 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 20
+    ny = 10
+    # nx = 10
+    # ny = 3
+    xmax = 10
+    ymax = 3
+    elem_type = TRI3
+  []
+  [pin]
+    type = ExtraNodesetGenerator
+    nodes = 106
+    new_boundary = pin
+    input = gen
+  []
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    strain = SMALL
+    incremental = false
+    add_variables = true
+    extra_vector_tags = 'kernel_residual'
+  []
+[]
+
+[BCs]
+  [left_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = left
+    value = 0
+  []
+  [left_y]
+    type = DirichletBC
+    variable = disp_y
+    boundary = pin
+    value = 0
+  []
+
+  inactive = 'right_dirichlet'
+  # inactive = 'right_neumann'
+  [right_neumann]
+    type = FunctionNeumannBC
+    variable = disp_x
+    function = t
+    boundary = right
+  []
+  [right_dirichlet]
+    type = FunctionDirichletBC
+    variable = disp_x
+    function = t/10
+    boundary = right
+  []
+[]
+
+[Materials]
+  [stress]
+    type = ComputeLinearElasticStress
+    # type = ComputeFiniteStrainElasticStress
+  []
+  [elasticity]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 100
+    poissons_ratio = 0.3
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
```

Comment:
Is this needed?

Reply:
I think it is. There are no kernels defined. All residual contributions come from the UEL plugin interface.

Reply:
D'Uh, you're right